pkg/proxy/apis/config/validation/validation_test.go

```diff
@@ -1026,7 +1005,7 @@ func TestValidateKubeProxyExcludeCIDRs(t *testing.T) {
 		},
 		{
 			addresses: []string{"2001:db8:xyz/64"},
-			msg:       "must be a valid IP block",
```

Comment:
is this change part of https://github.com/kubernetes/kubernetes/pull/95036 ?

Reply:
Yes